docs/docs/Components/components-data.md

```diff
@@ -361,16 +280,104 @@ This mode passes the webhook's data as a string for the **Chat Output** componen
 7. Open the **Playground**.
 Your JSON data is posted to the **Chat Output** component, which indicates that the webhook component is correctly triggering the flow.
 
-### Inputs
+<details>
+<summary>Parameters</summary>
+
+**Inputs**
 
 | Name | Display Name | Description |
 |------|--------------|-------------|
 | data | Payload | Receives a payload from external systems through HTTP POST requests. |
 | curl | cURL | The cURL command template for making requests to this webhook. |
 | endpoint | Endpoint | The endpoint URL where this webhook receives requests. |
 
-### Outputs
+**Outputs**
 
 | Name | Display Name | Description |
 |------|--------------|-------------|
 | output_data | Data | Outputs processed data from the webhook input, and returns an empty [Data](/concepts-objects) object if no input is provided. If the input is not valid JSON, the component wraps it in a `payload` object. |
+
+</details>
+
+## Legacy components
+
+Legacy components are available for use but are no longer supported.
+
+### Gmail Loader
+
+This component loads emails from Gmail using provided credentials and filters.
+
+For more on creating a service account JSON, see [Service Account JSON](https://developers.google.com/identity/protocols/oauth2/service-account).
```

Comment:
Per the Google style guide, use "about" instead of "on" in link intros. Apply this throughout the PR.